docroot/themes/humsci/su_humsci_theme/patterns/molecules/hero-text-overlay/scss/hero-text-overlay.scss

```diff
@@ -0,0 +1,267 @@
+@media (min-width: 1500px) {
+  .hero-text-overlay {
+    .image,
+    .image img {
+      height: 100%;
+      max-height: 760px;
+      object-fit: cover;
+      object-position: center center;
+      width: 100%;
+    }
+  }
+}
+
+.hero-text-overlay {
+  position: relative;
+
+  .image {
+    &:before {
+      bottom: 0;
+      content: '';
+      display: flex;
+      height: 100%;
+      position: absolute;
+      width: 100%;
+      z-index: 1;
+      background-image: linear-gradient(90deg, #00505c 14.95%, rgba(255, 255, 255, 0) 99.98%);
+    }
+
+    &.color-purple:before {
+      background-image: linear-gradient(90deg, #53284f 14.95%, rgba(255, 255, 255, 0) 99.98%);
+    }
+
+    &.color-paloalto:before {
+      background-image: linear-gradient(90deg, #175e54 14.95%, rgba(255, 255, 255, 0) 99.98%);
+    }
+
+    &.color-coolgrey:before {
+      background-image: linear-gradient(90deg, #4d4f53 14.95%, rgba(255, 255, 255, 0) 99.98%);
+    }
+
+    &.color-brown:before {
+      background-image: linear-gradient(90deg, #5e3032 14.95%, rgba(255, 255, 255, 0) 99.98%);
+    }
+  }
+
+  .overlay-text {
+    border: 1px solid #fff;
+    color: #fff;
+    font-size: 18px;
+    font-weight: 300;
+    left: 0;
+    line-height: 23px;
+    margin: 0 0 0 3em;
+    padding: 2em;
+    position: absolute;
+    top: 50%;
+    transform: translate(0%, -50%);
+    width: 35%;
+    z-index: 2;
+
+    h2,
+    .title h2 {
+      font-size: 35px;
+      font-weight: 300;
+      line-height: 42px;
+      margin-top: 0;
+    }
+
+    div.text-long,
+    p {
+      margin-bottom: 35px;
+    }
+  }
+}
+
+#main-content .hero-text-overlay .overlay-text {
+  a,
+  .link a,
+  .decanter-button {
+    background-color: transparent;
+    border: 1px solid #fff;
+    color: #fff;
+    font-weight: 400;
+    padding: 1rem 2rem 1.5rem;
+    text-decoration: none;
+
+    &:hover,
+    &:focus {
+      color: #fff;
+      text-decoration: none;
+      transition: background-color 0.5s ease;
+    }
+  }
+
+  &.color-default {
```

Comment:
I think it might be useful to have a localized mixin. https://sass-lang.com/documentation/at-rules/mixin#arguments